src/containers/Profile/index.js

```diff
@@ -4,59 +4,131 @@ import { createUser } from '../../thunks/createUser';
 import { connect } from 'react-redux';
 
 export class Profile extends Component {
-  constructor() {
-    super();
+  constructor(props) {
+    super(props);
     this.state = {
+      name: this.props.name || '',
+      email: this.props.email || '',
       program: '',
-      module: ''
-    }
+      module: '',
+      pronouns: '',
+      slack: '',
+      skill1: '',
+      skill2: ''
+    };
   }
 
   checkDropdowns = () => {
     const { program, module } = this.state;
     return !program || !module;
   };
 
-  handleOptionChange = event => {
+  handleChange = event => {
     let { value, name } = event.target;
     name = name.toLowerCase();
     this.setState({ [name]: value });
   };
 
-  handleSubmit = async (event) => {
+  handleSubmit = async event => {
     event.preventDefault();
-    const { name, email, image, firebaseID } = this.props;
-    const { module, program } = this.state;
-    const user = { name, email, image, firebaseID, module, program };
+    const { image, firebaseID } = this.props;
+    const {
+      module,
+      program,
+      name,
+      email,
+      pronouns,
+      slack,
+      skill1,
+      skill2
+    } = this.state;
+    const user = {
+      name,
+      email,
+      image,
+      firebaseID,
+      module,
+      program,
+      pronouns,
+      slack,
+      skill1,
+      skill2
+    };
     await this.props.createUser(user);
   };
 
   render() {
+    const { name, slack, email } = this.state;
+    const skills = [
+      'grid',
+      'flexbox',
+      'mythical creatures',
+      'react',
+      'jsFun',
+      'rspec',
+      'rails',
+      'ruby',
+      'jest/enzyme',
+      'docker',
+      'lightning talks',
+      'javascript',
+      'css',
+      'active record',
+      'SQL',
+      'sinatra'
+    ];
     return (
       <div>
+        <label htmlFor='name'>Provide your name</label>
+        <input value={name} name='name' onChange={this.handleChange} />
+        <label htmlFor='email'>Provide your email handle</label>
+        <input value={email} name='email' onChange={this.handleChange} />
+        <label htmlFor='slack'>Provide your slack handle</label>
+        <input value={slack} name='slack' onChange={this.handleChange} />
+        <Dropdown
```

Comment:
Great job reusing the Dropdown component for gathering this information for each user!🔥